gemma-core/src/main/java/ubic/gemma/model/expression/arrayDesign/ArrayDesignValueObject.java

```diff
@@ -161,40 +179,6 @@ public ArrayDesignValueObject( Long id ) {
         super( id );
     }
 
-    /**
-     * Minimal constructor
-     * 
-     * @param id        the id
-     * @param shortName the shortName
-     * @param name      the name
-     */
-    public ArrayDesignValueObject( Long id, String shortName, String name ) {
-        super();
-        this.id = id;
-        this.shortName = shortName;
-        this.name = name;
-    }
-
-    public ArrayDesignValueObject( Object[] row, Integer totalInBatch ) {
-        super( ( Long ) row[0], ( Date ) row[6], ( Boolean ) row[7], ( AuditEvent ) row[13], ( Boolean ) row[8],
-                ( AuditEvent ) row[12], ( String ) row[9], ( AuditEvent ) row[11], totalInBatch );
-
-        this.name = ( String ) row[1];
-        this.shortName = ( String ) row[2];
-
-        TechnologyType c = ( TechnologyType ) row[3];
-        if ( c != null ) {
-            this.technologyType = c.toString();
-            this.color = c.getValue();
-        }
-
-        this.description = ( String ) row[4];
-        this.isMergee = row[5] != null;
-        this.isAffymetrixAltCdf = row[14] != null;
-
-        this.taxon = ( String ) row[10];
-    }
-
```

Comment:
This is the kind of changes I really like to introduce. With entity-oriented HQL, we delegate all this to Hibernate.